File: project 2 - loop/php/Rock_Paper_Scissors.php
<?php
$choices = array("rock", "paper", "scissors");

$userChoice = isset($_GET['choice']) ? strtolower($_GET['choice']) : '';

$computerChoice = $choices[array_rand($choices)];

if (!in_array($userChoice, $choices)) {
    echo "Invalid choice: $userChoice";
} else {
    echo "Computer's choice: $computerChoice\n";
    echo "Your choice: $userChoice\n";

    if ($userChoice == $computerChoice) {
        echo "It's a tie!";
    } elseif (
        ($computerChoice == "rock" && $userChoice == "scissors") ||
        ($computerChoice == "scissors" && $userChoice == "paper") ||
        ($computerChoice == "paper" && $userChoice == "rock")
    ) {
        echo "Computer wins!";
    } else {
        echo "You win!";
    }
}
?>

File: project 2 - loop/python/Rock_Paper_Scissors.py
import random
while True:
    choices = ["rock","paper","scissors"]
    cumpoterChoice = random.choice(choices)
    userChoice = input("enter your choice").lower()
    
    if userChoice not in choices:
        print(f"invalid {userChoice}")
    else:
        print(f"cumpoterChoice = {cumpoterChoice}")
        print(f"your Choice = {userChoice}")
    
        if cumpoterChoice == userChoice:
            print("equal")
        elif (cumpoterChoice == "rock" and userChoice == "scissors") or\
        (cumpoterChoice == "scissors" and userChoice == "papaer") or\
        (cumpoterChoice == "paper" and userChoice == "rock") :
            print("cumpoter win")
        else:
            print("you win")    
                
